airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/initialsync/MssqlInitialReadUtil.java

```diff
@@ -246,16 +246,18 @@ public static Map<io.airbyte.protocol.models.AirbyteStreamNameNamespacePair, Ord
     return pairToOcInfoMap;
   }
 
-  final static OrderedColumnInfo getOrderedColumnInfo(final JdbcDatabase database,
-                                                      final ConfiguredAirbyteStream stream,
-                                                      final Map<String, TableInfo<CommonField<JDBCType>>> tableNameToTable,
-                                                      final String quoteString) {
+  static OrderedColumnInfo getOrderedColumnInfo(final JdbcDatabase database,
+                                                final ConfiguredAirbyteStream stream,
+                                                final Map<String, TableInfo<CommonField<JDBCType>>> tableNameToTable,
+                                                final String quoteString) {
     // For cursor-based syncs, we cannot always assume a ordered column field exists. We need to handle
     // the case where it does not exist when we support cursor-based syncs.
     if (stream.getStream().getSourceDefinedPrimaryKey().size() > 1) { // TODO: validate the seleted column rather than primary key
```

Comment:
@xiaohansong I don't think `getSourceDefinedPrimaryKey()` is correct here, is it?